content/java/concepts/array-list/terms/addall/addall.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: '.addAll()'
+Description: 'Adds a collection to an ArrayList.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.addAll()` method is used to add the contents of a collection to an instance of the [`ArrayList`](https://www.codecademy.com/resources/docs/java/array-list) class.
```

Comment:
```suggestion
The **`.addAll()`** method is used to add the contents of a collection to an instance of the [`ArrayList`](https://www.codecademy.com/resources/docs/java/array-list) class.
```

One more

Reply:
😆 How bold 😬 +1 for consistency, going to start bolding keywords as well